examples/IBFE/explicit/ex5/example.cpp

```diff
@@ -344,17 +346,27 @@ main(int argc, char* argv[])
         }
         Pointer<IBStrategy> ib_ops;
         if (use_boundary_mesh)
+	{
             ib_ops = new IBFESurfaceMethod(
                 "IBFEMethod",
                 app_initializer->getComponentDatabase("IBFEMethod"),
                 &mesh,
-                app_initializer->getComponentDatabase("GriddingAlgorithm")->getInteger("max_levels"));
+                app_initializer->getComponentDatabase("GriddingAlgorithm")->getInteger("max_levels"),
+                /*register_for_restart*/ true,
+		restart_read_dirname,
+		restart_restore_num);
```

Comment:
It looks like here, and elsewhere, you are mixing tabs with spaces. Do you know how to run clang-format on your recent changes?

Reply:
no

Reply:
does it need to be all spaces?

Reply:
Yes - running clang-format will do this for you automatically.